pkg/kube/snapshot/mocks/mock_snapshotter.go

```diff
@@ -1,8 +1,8 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: github.com/kanisterio/kanister/pkg/kube/snapshot (interfaces: Snapshotter)
+// Source: pkg/kube/snapshot/snapshot.go
 
-// Package mocks is a generated GoMock package.
-package mocks
+// Package mock_snapshot is a generated GoMock package.
+package mock_snapshot
```

Comment:
How is it generated? Can we keep the package name same?

Reply:
This was generated using `mockgen`. Updated the package name.